### DISCUSSION TITLE
Fody support for DynamicData Bind

### DISCUSSION THREAD

**Discussion by martinbu** [reactiveui/ReactiveUI#3691] [General]
This question was already asked here https://github.com/reactivemarbles/DynamicData/issues/348
But I couldn't find any answer if it is possible or already implemented but not documented...

Can anyone tell me if there is a Fody Attribute for generating the private field for the dynamic data bind use case?

**Comment by glennawatson** [comment]
No. Fody support ongoing is not planned and definitely not to expanding to include dynamicdata. Potentially a good use case for source generators though.

**Comment by martinbu** [comment]
Implemented by: https://github.com/reactiveui/ReactiveUI.SourceGenerators/pull/180
